CRM/Donrec/Form/Task/DonrecTask.php

```diff
@@ -104,14 +104,24 @@ function postProcess() {
 
     if (!empty($result['intersection_error'])) {
       CRM_Core_Session::singleton()->pushUserContext(
-        CRM_Utils_System::url('civicrm/donrec/task', 'conflict=1' . '&sid=' . $result['snapshot']->getId() . '&ccount=' . count($this->_contactIds)));
+        CRM_Utils_System::url(
+          'civicrm/donrec/task',
+          'conflict=1'
+          . '&sid=' . $result['snapshot']->getId()
+          . '&ccount=' . count($this->getContactIDs())
+        )
+      );
     }elseif (empty($result['snapshot'])) {
       CRM_Core_Session::setStatus(E::ts('There are no selectable contributions for these contacts in the selected time period.'), E::ts('Warning'), 'warning');
       $qfKey = $values['qfKey'];
       CRM_Utils_System::redirect(CRM_Utils_System::url('civicrm/contact/search', "_qf_DonrecTask_display=true&qfKey=$qfKey"));
     }else{
       CRM_Core_Session::singleton()->pushUserContext(
-        CRM_Utils_System::url('civicrm/donrec/task', 'sid=' . $result['snapshot']->getId() . '&ccount=' . count($this->_contactIds))
+        CRM_Utils_System::url(
+          'civicrm/donrec/task',
+          'sid=' . $result['snapshot']->getId()
+          . '&ccount=' . count($this - $this->getContactIDs())
```

Comment:
@jensschuppe I think this calculation is not intended and will cause a type error in PHP 8.

Reply:
Thanks for spotting that, I released 2.2.3 with that being fixed.